qiita_db/util.py

```diff
@@ -656,6 +656,9 @@ def retrieve_filepaths(obj_fp_table, obj_id_column, obj_id):
         The name of the column that represents the object id
     obj_id : int
         The object id
+    sort : {'ascendent', 'descendent'}, optional
```

Comment:
Rather have  `{'ascendent', 'descendent'}` or `{'ascending', 'descending'}`?

[pandas](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.sort.html) uses the later.